docs/public-api.asciidoc

```diff
@@ -211,6 +211,63 @@ var transaction2 = Agent.Tracer.StartTransaction("Transaction2", "TestTransactio
      DistributedTracingData.TryDeserializeFromString(serializedDistributedTracingData));
 ----
 
+[float]
+[[api-start-capture-error]]
+==== `void CaptureError(string message, string culprit, StackFrame[] frames = null, string parentId = null);`
+Use this method to capture an APM error with a message and a culprit.
+
+NOTE: If there is an active transaction, the captured error will be correlated to it, otherwise the error will show up, but it won't be correlated to a transaction on the APM UI.
```

Comment:
For the three identical NOTEs on this page, how do you feel about rewording to something like this?

```
Captured errors are automatically correlated with the active transaction. If no transaction is active, the error will still appear in the APM app but will not be correlated with a transaction.
```

Reply:
Thanks, yes, sounds better. I updated to this one.